lib/OpenQA/UserAgent.pm

```diff
@@ -30,9 +30,13 @@ sub new {
             for my $i (qw(key secret)) {
                 my $attr = "api$i";
                 next if $self->$attr;
-                my $val = $cfg->val($args{api}, $i);
+                # Fetch all the values in the file.
+                my @values = $cfg->val($args{api}, $i);
+                # Keep the latest one
+                my $val = pop @values;
                 next unless $val;
-                $val =~ s/\s+$//;    # remove trailing whitespace
+                # remove trailing whitespace
```

Comment:
Why moving the comment into its own line? This comment is not very important. It is just there because not everybody who might read this is very good at deciphering regexes.

Reply:
Is there a style guide to follow? I just followed my instinct on this.

Addressed in 5b58740.